### PR TITLE
add Resizable UI feature for beta testers

### PR DESCRIPTION
**Pull request by lol-catz**
Howdy Ho...  Ive seen a few people on discord request this,,,

### CODE REVIEW

File: source/jucePlugin/PluginEditor.cpp [hunk 1]
```diff
@@ -14,6 +14,9 @@ AudioPluginAudioProcessorEditor::AudioPluginAudioProcessorEditor(AudioPluginAudi
     // editor's size to whatever you need it to be.
 	setSize(800, 400);
 
+	// Resizable UI
```
Comment (talaviram) [inline source/jucePlugin/PluginEditor.cpp:17]: I know this is temporary. But if already adding this. IMHO worth adding resize limits.

Comment (lol-catz) [reply source/jucePlugin/PluginEditor.cpp:17]: agree..

setResizeLimits(800,400,800,1600);  

sound reasonable to you?

Comment (lol-catz) [reply source/jucePlugin/PluginEditor.cpp:17]: (ill add spaces after the comma's of course, lol)

Comment (talaviram) [reply source/jucePlugin/PluginEditor.cpp:17]: Feel free to make even 4k Max size

Comment (lol-catz) [reply source/jucePlugin/PluginEditor.cpp:17]: i think 1600 is plenty...  its a temporary hack..  more the 4x the parameters are now available at one time without scrolling..   just a quick and easy quality of life improvement for the beta testers...  done!

**Pull request closed without merging**